openshift/release/artifacts/eventing-kafka-controller.yaml

```diff
@@ -1137,6 +1137,14 @@ metadata:
 rules:
   - apiGroups:
       - "*"
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
```

Comment:
@pierDipi is this valid/needed for 1.6 too?

Reply:
Fixed